scss/editor/_layout.scss

```diff
@@ -181,6 +182,26 @@
         }
     }
 
+    .k-editor-table-wizard-dialog {
+        .k-content {
+            padding: $spacer-y $padding-x 0;
```

Comment:
`$spacer` should not be used for padding, rather only for margin. `$padding-x` is usually sufficient enough.

Not to mention that `$spacer` uses `em` and `$padding` uses `px`.